packages/react-bootstrap-table2/src/bootstrap-table.js

```diff
@@ -131,6 +134,10 @@ BootstrapTable.propTypes = {
   bordered: PropTypes.bool,
   hover: PropTypes.bool,
   condensed: PropTypes.bool,
+  tableCaption: PropTypes.oneOfType([
```

Comment:
I think this props name can be more simple like `caption`, I think it's enough and easy to understand for developer.